tests/ui/js/transformsTests.js

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 OCAD University
+Copyright 2017-2019 OCAD University
```

Comment:
Is 2017 the correct start date? You may also want to transition this repo, in a future PR,  to the centralized Copyright like we now have in Infusion.

Reply:
2017 is indeed the correct start date. At one point in 2018, I updated all the copyright dates to 2018 rather than indicate a range, so I've been fixing these here and there as I see them.

Reply:
[SJRK-267](https://issues.fluidproject.org/browse/SJRK-267) has been filed to move to a centralized copyright